### ISSUE TITLE
Dropout rates to be updated in all ALBERT v2 configs

### ISSUE THREAD

**Issue by matteodelv** [huggingface/transformers#2337]
## 🐛 Bug

Model I am using (Bert, XLNet....): ALBERT v2

Language I am using the model on (English, Chinese....): English

The problem arise when using:
* [x] the official example scripts: run_squad.py
* [ ] my own modified scripts: 

The tasks I am working on is:
* [x] an official GLUE/SQUaD task: SQuAD v2.0
* [ ] my own task or dataset: 

## To Reproduce

1. Just choose one of the following default ALBERT v2 configs: base, large, xlarge

## Expected behavior

As stated in the updated ALBERT Tensorflow repo from Google Research, model v2 introduces no dropout at all for downstream tasks, like SQuAD, MRPC or COLA.
Following the discussing from this [issue](https://github.com/google-research/ALBERT/issues/23), model configurations on TF-Hub were wrong and thus, the ones used by transformers (loaded on Amazon S3).

<s>While configs on TF-Hub will be updated in the near future</s> As configs on TF-Hub have already been updated, transformers' ones should be updated too: parameters `attention_probs_dropout_prob` and `hidden_dropout_prob` should be both 0 for all v2 configs.

## Environment

* OS: Platform Linux-4.14.152-98.182.amzn1.x86_64-x86_64-with-glibc2.9
* Python version: Python 3.6.5 |Anaconda, Inc.| (default, Apr 29 2018, 16:14:56)
* PyTorch version: PyTorch 1.3.1
* PyTorch Transformers version (or branch):
* Using GPU ? YES, 4x NVIDIA V100 16GB
* Distributed of parallel setup ? No
* Any other relevant information: Using Amazon AWS Deep Learning Linux AMI 26.0

## Additional context

Hope this will be useful!

**Comment by tbright17** [comment]
I found it already been updated: https://s3.amazonaws.com/models.huggingface.co/bert/albert-xxlarge-v2-config.json

Did I miss something?

**Comment by matteodelv** [comment]
Base, large and xlarge v2 configs have to be updated too, as confirmed by this [issue](https://github.com/google-research/ALBERT/issues/23) in the official Google Research repository.

**Comment by tbright17** [comment]
Thanks. Will pay attention to this one. Hope it will be fix soon.
